vars/RedHat_10.yml

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: MIT
+---
+__snapshot_packages: ["lvm2", "util-linux", "snapm", "boom-boot"]
```

Comment:
do you need `boom-boot` now?  I would prefer to wait to add this package when boom-boot support is added

Reply:
yes, boom-boot is required by snapm.